### PR TITLE
Docs: Started Russian translation.

### PR DESCRIPTION
**Pull request by bad4iz**
localization of documentation into Russian

translation of the Manual section into Russian

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

File: docs/manual/ru/introduction/Installation.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	<p>
+		Если three.js был установлен с CDN, используйте тот же CDN для установки других компонентов:
+	</p>
+
+	<code>
+		&lt;script type="module">
+
+		import { OrbitControls } from 'https://unpkg.com/three@&lt;version&gt;/examples/jsm/controls/OrbitControls.js';
+
+		const controls = new OrbitControls( camera, renderer.domElement );
```
Comment (Mugen87) [inline docs/manual/ru/introduction/Installation.html:154]: One thing: This section has recently changed. An import map was added and the term `examples` was renamed to `addons` (see #24887).

Comment (bad4iz) [reply docs/manual/ru/introduction/Installation.html:151]: I corrected

File: docs/manual/ru/introduction/Installation.html [hunk 2]
```diff
@@ -143,22 +139,23 @@ <h2>Примеры</h2>
...
 	<p>
-		Если three.js был установлен с CDN, используйте тот же CDN для установки других компонентов:
+		Если three.js был установлен с CDN, используйте тот же код, но с `three/addons/` в карте импорта.
 	</p>
 
 	<code>
 		&lt;script type="module">
 
-		import { OrbitControls } from 'https://unpkg.com/three@&lt;version&gt;/examples/jsm/controls/OrbitControls.js';
+		import * as THREE from 'three';
```
Comment (Mugen87) [inline docs/manual/ru/introduction/Installation.html:148]: I think we also need an import map here.

Comment (bad4iz) [reply docs/manual/ru/introduction/Installation.html]: thanks for the comment. corrected it.

**Pull request merged**